Version.d.tpl

```diff
@@ -46,6 +46,8 @@ static this()
     version_info["build_date"] = "@DATE@";
     version_info["build_author"] = "@AUTHOR@";
     version_info["compiler"] = "@COMPILER@";
+    version_info["dflags"] = "@DFLAGS@";
```

Comment:
This may be confusing as it won't include flags supplied to DMD indirectly via `RDMDFLAGS` or some other env var. Is that OK?

Reply:
Yeah, I think including `RDMDFLAGS` is more noise than anything else, because it's all about the build system, not how code is actually compiled (unless `RDMDFLAGS` are misused to pass `DFLAGS`). Currently `RDMDFLAGS` passed automatically are `--compiler` (already present in `Version`) and `--force` (irrelevant).